layouts/_default/landing.html

```diff
@@ -0,0 +1,192 @@
+{{ define "body-class" }}
+  text-[16px] bg-beige font-serif leading-normal text-s-9
+{{ end }}
+{{ define "html-class" -}}
+  scroll-smooth
+{{- end }}
+{{ define "static-head" }}
+  {{ partialCached "header/static-tw.html" . }}
+{{ end }}
+
+{{ define "header" }}
+  <header data-ga-category="header">
+    {{ block "nav" . }}
+      {{ partialCached "tw/nav.html" . }}
+    {{ end }}
+    {{ block "banner" . }}
+      {{ if .Param "banner-active" }}
+        {{ partialCached "tw/banner.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "topper" . }}
+      {{ if .Param "topper-active" }}
+        {{ partialCached "tw/promo-topper.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "promo" . }}
+      {{ partialCached "tw/promo-header.html" . }}
+    {{ end }}
+  </header>
+{{ end }}
+
+{{ define "main" }}
+  <article class="bg-white px-4 pt-12 pb-2 md:pb-6 overflow-visible">
+    <header>
+      <div class="mx-auto w-full max-w-screen-md">
+        <h1
+          class="font-sans text-2xl sm:text-4xl md:text-5xl font-black leading-none tracking-wide"
+        >
+          {{- .Title -}}
+        </h1>
+      </div>
+      <div class="mx-auto w-full max-w-prose">
+
+        {{ partial "tw/image-hero.html" . }}
+
+      </div>
+    </header>
+    {{ with .Content }}
+      <div class="mx-auto w-full max-w-screen-md">
+        <div class="mt-6" data-ga-category="content">
+          <div
+            x-data
+            x-intersect.once="$report({ target: $el })"
+            data-ga-action="view:content-start"
+          ></div>
+          <div
+            class="article-content max-w-prose w-full overflow-visible leading-relaxed"
+            data-sf-campaign="701f4000000TVuXAAW"
+          >
+            {{ . }}
+          </div>
+          <div
+            x-data
+            x-intersect.once="$report({ target: $el })"
+            data-ga-action="view:content-finish"
+          ></div>
+        </div>
+      </div>
+    {{ end }}
+
+  </article>
+
+  <section class="min-h-[65vh] bg-black bg-opacity-10 py-20">
+    <div class="mx-auto max-w-6xl px-5 lg:flex">
+      <h2 
+        class="lg:sideways-heading mb-4 text-4xl font-black font-sans uppercase leading-none text-s-7 opacity-40 md:text-6xl lg:mb-0 lg:rotate-180"
+      >
+        Benefits
+      </h2>
+      <div class="flex-[0_1_46%] flex flex-row flex-wrap max-w-5xl text-lg font-sans text-s lg:pl-20">
+        <h3 class="flex-[1_1_auto] self-end font-black font-sans text-3xl text-s">Private <br>Events</h3>
+        <svg class="w-20 -order-1 mr-6 self-end" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 82 82"><g fill="#000" fill-rule="evenodd"><path d="m0 54.917 6.77 6.77A9.575 9.575 0 1 1 20.312 75.23L27.084 82 65 44.083 37.917 17 0 54.917ZM41 13.667 68.333 41 82 27.333 75.167 20.5c-3.775 3.774-9.894 3.774-13.667 0-3.774-3.774-3.774-9.893 0-13.667L54.666 0 41 13.667Z"/></g></svg>
```

Comment:
Where did this SVG come from?

Reply:
These came from The Noun Project. We will need to credit them if we decide to use them. I can also create custom icons fairly quickly